packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -80,7 +80,9 @@ function renameSupport (lang: 'js' | 'ts' | 'coffee' = 'js') {
 }
 
 describe('global mode', () => {
-  it('migrates 2 projects in global mode', () => {
+  // TODO: Figure out why it is flaky. Seems to be due to MigrationWizard query being executed multiple times
+  // see: https://github.com/cypress-io/cypress/issues/25377
+  it.skip('migrates 2 projects in global mode', () => {
```

Comment:
The linked issue was closed 3 days ago, should this still be skipped? If  so the issue should prob be reopened https://github.com/cypress-io/cypress/issues/25377

Reply:
Unskipped test. Looks like more branching mess. :+1: